### DISCUSSION TITLE
URL Data Type

### DISCUSSION THREAD

**Discussion by ghislaineguerin** [mathesar-foundation/mathesar#544] [Maintainer discussions]
The specs related to issue #236 are ready for review. Please take a look at the attached document and follow the [review guidelines](https://wiki.mathesar.org/en/design/process/review-guidelines).

## Document Link
https://wiki.mathesar.org/en/design/specs/url-data-type

@mathemancer @kgodey @pavish, your feedback is required by Wednesday 18th. Please plan your review so that at least one round of changes can be made and reviewed if needed.

**Comment by kgodey** [comment]
@ghislaineguerin Looks good! Feedback:

- URLs should be clickable. We need to format them like links so that users know they are clickable.
- It would be good to have a list of all URL filtering options somewhere, either in the prototype or the spec text document. The frontend needs to have a full list of what to implement.
- We need to have error handling for invalid URL formats

**Reply by ghislaineguerin** [reply]
@kgodey Thanks. I have updated based on your feedback.

**Reply by kgodey** [reply]
Looks good to me!

**Comment by ghislaineguerin** [comment]
@kgodey @pavish The spec document has been updated to include the global data type component.

**Reply by kgodey** [reply]
Looks good, thanks!
